### PR TITLE
Port GLDebug mappings to latest

### PR DESCRIPTION
**Pull request by RDIL**
This is all I have time for right now sadly, but here you go!

### CODE REVIEW

**Review by liach — APPROVED** [review]
That indent style stuff looks good, guess

**Comment by liach** [comment]
Please edit the pr to merge into `1.16-pre7`.

**Comment by RDIL** [comment]
Done.

**Comment by liach** [comment]
Unfortunately pre8 is out

**Comment by liach** [comment]
Mind merge into 1.16-rc1 instead

**Comment by RDIL** [comment]
You drive a hard bargain.  Done.

File: .editorconfig [hunk 1]
```diff
@@ -1,5 +1,2 @@
-[*.gradle]
-indent_style = tab
-
-[*.java]
+[*.{gradle,mapping}]
```
Comment (YanisBft) [inline .editorconfig:1]: Why not ` indent_size = 4` as well like in your 1.14.4 pr?

Comment (RDIL) [reply .editorconfig:1]: Its the default, it was late at night for me when I wrote the ones for 1.14 😓

**Review by YanisBft — APPROVED** [review]

**Pull request merged**